gui/src/gui/ssh.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, *args, **kwargs):
         self.stack.add_named(self.preferences_page, "preferences_page")
 
         self.add_ssh_key_group = Adw.PreferencesGroup()
-        self.add_ssh_key_group.set_title("Add an SSH Key")
+        self.add_ssh_key_group.set_title("Add a SSH Key")
```

Comment:
"an SSH Key" is actually correct here, because it's pronounced ess-ess-aitch.

Reply:
Ahh, I see. I've reverted this change.